src/pykiso/lib/robot_framework/communication_auxiliary.py

```diff
@@ -60,16 +60,29 @@ def send_message(
         logger.info(f"send message {raw_msg} using {aux_alias}")
         return state
 
```

Comment:
So, no start / stop recording in robot_framework?

Reply:
I think robot framework have first to be fix, and then we could implement the start and stop recording :)

Reply:
What needs to be fixed?